src/main.rs

```diff
@@ -279,15 +279,15 @@ fn push_impl(repo: &Repository, matches: &clap::ArgMatches) -> i32 {
         // push a specific list of issues
         issues.map(Oid::from_str).abort_on_err()
               .map(|issue| repo.get_issue_refs(issue)).abort_on_err()
-              .flat_map(|refs| refs).abort_on_err()
-              .filter_map(|issue_ref| remote.issue_refspec_from_ref(issue_ref))
+              .flat_map(git2::References::names).abort_on_err()
```

Comment:
I would put the `abort_on_err()` on the next line. It is better for readability.